tests/languages/tt2/variable_feature.test

```diff
@@ -0,0 +1,36 @@
+[% 
+	foo
+	foo.bar
+	foo.2
+	foo.1.bar.2.baz.3
+	foo.$index
+	foo
+		.
+			bar
+	world
+	hands
+	knots
+%]

```

Comment:
Could you add a test for the `Project::Specials` kind of variables here? I know it's technically already tested in the test for keywords, but since it's handled by the `variable` pattern, it definitely belongs here too.

Reply:
Looks like it's not supported anyway, see above.  So I'll remove the colons from the variable regex as well.